omnibus/config/software/datadog-agent-integrations-py2.rb

```diff
@@ -142,7 +142,7 @@
     #
     command "#{pip} download --dest #{build_deps_dir} hatchling==0.25.1", :env => pre_build_env
     command "#{pip} download --dest #{build_deps_dir} setuptools==40.9.0", :env => pre_build_env # Version from ./setuptools2.rb
-    command "#{pip} install wheel==0.34.1", :env => pre_build_env
+    command "#{pip} install wheel==0.38.4", :env => pre_build_env
```

Comment:
:/ they dropped py2 support with `0.38.0` https://github.com/pypa/wheel/blob/main/docs/news.rst. Having a look

Reply:
Seems like the best we can do here is upgrading to `0.37.1`

Reply:
Reverted for py2

Reply:
Set to 0.37.1